telemetry/definitions/commonDefinitions.json

```diff
@@ -1457,6 +1497,31 @@
             "type": "int",
             "description": "Number of generations before the user accepted or rejected"
         },
+        {
+            "name": "numberOfBuildExecuted",
+            "type": "int",
+            "description": "Number of build executed in unit test generation build loop"
+        },
+        {
+            "name": "numberOfBuildFailed",
+            "type": "int",
+            "description": "Number of build failures in unit test generation build loop"
+        },
+        {
+            "name": "numberOfBuildPassed",
+            "type": "int",
+            "description": "Number of build succeeded in unit test generation build loop"
+        },
+        {
+            "name": "numberOfUnitTestCasesAccepted",
+            "type": "int",
+            "description": "Number of unit test cases accepted"
+        },
+        {
+            "name": "numberOfUnitTestCasesGenerated",
+            "type": "int",
```

Comment:
use or create generic `fooCount` metrics.

please always do this in the future. I have mentiond this in previous reviews.

Reply:
Named these two fields to common`acceptedCount` and `generatedCount`.